README.md

```diff
@@ -8,26 +8,96 @@ The goal of this project is to make a feature complete Gothic game client compat
 [![Latest build](https://img.shields.io/github/release-pre/Try/opengothic?style=for-the-badge)](https://github.com/Try/opengothic/releases/latest)
 ![Screenshoot](scr0.png)
 
-### Work in progress
+#### Work in progress
 [![Build status](https://ci.appveyor.com/api/projects/status/github/Try/opengothic?svg=true)](https://ci.appveyor.com/project/Try/opengothic)
 
-The original game has been replicated fully; you can complete both the main quest and the addon.
-Check out the [bugtracker](https://github.com/Try/OpenGothic/issues) for a list of known issues.
+The original game has been replicated fully; you can complete both the main quest and the addon. OpenGothic runs reasonably well except on Low-End devices such as Raspberry-Pi. Check out the [bugtracker](https://github.com/Try/OpenGothic/issues) for a list of known issues.
 
-### Install on Windows
-1. Install original Gothic game from CD/Steam/GOG/etc.
-*you have to install the original game as OpenGothic does not provide any built-in game assets or game scripts*
-2. [Download latest stable build](https://github.com/Try/opengothic/releases/latest)
-3. Run '/OpenGothic/bin/Gothic2Notr.exe -g "C:\Program Files (x86)\Path\To\Gothic II"'
+Development is focused on Gothic 2 and new features are not tested for Gothic 1 compatibility. While Gothic 1 is not officially supported pull requests that fix Gothic 1 — and general — bugs are welcome.
 
-Common Gothic installation paths to be provided via `-g`:
-- "C:\Program Files (x86)\JoWooD\Gothic II"
-- "C:\Gothic II"
-- "C:\Program Files (x86)\Steam\steamapps\common\Gothic II"
-- "~/PlayOnLinux's virtual drives/Gothic2_gog/drive_c/Gothic II"
+#### Prerequisites
 
-### Build on Linux
-#### Dependencies
+Gothic 2: Night of the Raven is required as OpenGothic does not provide any built-in game assets or scripts.
+
+Supported systems are:
+* Windows (DX12, Vulkan)
+* Linux (Vulkan)
+* MacOS (Metal)
+
+## How to play
+### Windows
+1. If not already done install Gothic. OpenGothic comes with auto-path detection if your Gothic files are in a common path.
+    * "C:\Program Files (x86)\JoWooD\Gothic II"
```

Comment:
`installdetect.cpp` seems to check only for steamapps and Akella Games (also not entirely sure which Program Files path is actually checked by OpenGothic on 64-bit systems)

as an aside beyond the scope of this PR, checking for default GOG install path would be nice too